src/components/common/NameRequestInfo.vue

```diff
@@ -91,7 +91,7 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
 
   /** The entity title  */
   private entityTypeDescription (): string {
-    return this.getCorpTypeDescription(this.parsedNameRequest.entityType)
+    return this.getCorpTypeDescription(this.parsedNameRequest.entityType as CorpTypeCd)
```

Comment:
This was OK previously, so it's either:
- a change in BCRS Shared Component
- an improvement in lint / Vue tools

Reply:
I've noticed this as well